docs/package/gluon-hoodselector.rst

```diff
@@ -0,0 +1,130 @@
+gluon-hoodselector
+==================
+
+This package provides an automated way to continuously select the correct
+domain based on the geolocation of a node. The purpose of Hoodselector is to
+automatically detect in which domain the node is located based on its
+geolocation settings. Therefore domains are required to have bounding boxes,
+defined as polygons or rectangles. Based on this information Hoodselector will
+select a domain from the list of known domains and migrate towards it without
+requiring a reboot. This package therefore provides a scalable and
+decentralized approach to automatic domain selection.
+
+Background informations
```

Comment:
Background information

I do the same mistake all the time, but I see it when someone else does.
Information is uncountable in English.

Reply:
fixed